src/app/api/label-mapping/label-mapping.service.ts

```diff
@@ -0,0 +1,168 @@
+import { AssigneeType } from '@prisma/client'
+import { BaseService } from '../core/services/base.service'
+import { CopilotAPI } from '@/utils/CopilotAPI'
+import { z } from 'zod'
+import { LabelMapping as PrismaLabelMapping } from '@prisma/client'
+
+export class LabelMapping extends BaseService {
+  /**
+   * This method returns the computed label
+   * @param assigneeId
+   * @param assigneeType
+   * @returns string
```

Comment:
I see all the `return` statement inside `if` conditions so there a possibility that this function might return `void`